reana/reana_dev/git.py

```diff
@@ -349,12 +355,14 @@ def git_clone(user, component):  # noqa: D301
                                to include several runable REANA demo examples;
                          * (7) special value 'ALL' that will expand to include
                                all REANA repositories.
-    :param user: The GitHub user name. [default=anonymous]
-    :type component: str
+    :param exclude_components: List of components to exclude from command.
     :type user: str
-
+    :type component: str
+    :type exclude_components: str
     """
-    components = select_components(component)
+    if exclude_components:
+        exclude_components = exclude_components.split(",")
+    components = select_components(component, exclude_components)
```

Comment:
Minor: What do you think about adding a common decorator like, `@click_add_exclude_components_option`, with a callback that performs this logic, [like we do for tokens in R-Client](https://github.com/reanahub/reana-client/blob/2436dde2df70ca835afcc49ad81f1f91ce79e030/reana_client/cli/utils.py#L23-L37)?

We could even have a single option decorator for components `@click_add_components_option` with the possibility of taking a parameter `allow_exclude_components=True` so components will always be the result of select components with or without exclude components filter.

```python
@git_commands.command(name="git-fork")
@click_add_components_option
def git_fork(components, ...):
    ...
@git_commands.command(name="git-status")
@click_add_components_option(allow_exclude_components=True)
def git_status(components, ...): 
    ...
```